src/libPMacc/include/algorithms/math/doubleMath/trigo.tpp

```diff
@@ -46,6 +46,17 @@ struct Sin<double>
 };
 
 template<>
+struct ASin<double>
+{
+    typedef double result;
+
+    HDINLINE double operator( )(const double& value )
```

Comment:
Space is missing after the parentheses. `HDINLINE double operator( )( const....)`